### DISCUSSION TITLE
如何讓 Sequelize 返回正確的時間格式給前端？（方法分享）

### DISCUSSION THREAD

**Discussion by KW0409** [Lidemy/forum#226] [靈感乍現]
使用 Sequelize 時，得到的時間格式總是長得奇形怪狀
在這邊分享兩個網路上找到的比較常見的解決方式
（兩個方法都還蠻簡易的，連結裡面也都有附上操作說明，所以這邊就不贅述了～）

**方法1：(修改 sequelize 的 config.json 設定)**
[使用sequelize操作数据库 时间格式化](https://segmentfault.com/a/1190000020009630)

**方法2：(利用 moment 套件)**
[sequlize查询结果时间格式化](https://blog.csdn.net/foreverling_ling/article/details/88694294)

P.S.
（個人用的是 方法1，畢竟我是事後修改，使用 方法1 比較方便，如果是 方法2 的話，感覺上是需要再重建一次 table）

**Comment by aszx87410** [comment]
2 不需要重建 table 喔，他只是去自定義拿值的時候要多做一層轉換，所以不需要動到 DB

**Reply by KW0409** [reply]
原來如此，感謝 Huli 老師的補充！
